zetaclient/testutils/constant.go

```diff
@@ -39,12 +39,14 @@ const (
 	EventERC20Withdraw = "Withdrawn"
 )
 
+// OldSolanaGatewayAddressDevnet is the old gateway address deployed on Solana devnet
```

Comment:
Can we explain why do we need it?

Reply:
We have a few (four) early transactions used by unit tests. Those transactions were performed on the early program ID `94U5AHQMKkV5txNJ17QPXWoh474PheGou6cNP2FEuL1d` in Devnet, for example: 

`zetaclient/testdata/solana/chain_901_inbound_tx_result_MS3MPLN7hkbyCZFwKqXcg8fmEvQMD74fN6Ps2LSWXJoRxPW5ehaxBorK9q1JFVbqnAvu9jXm6ertj7kT7HpYw1j`
`zetaclient/testdata/solana/chain_901_outbound_tx_result_5iBYjBYCphzjHKfmPwddMWpV2RNssmzk9Z8NNmV9Rei71pZKBTEVdkmUeyXfn7eWbV8932uSsPfBxgA7UgERNTvq`